python/pysmurf/client/base/smurf_control.py

```diff
@@ -180,6 +181,12 @@ def initialize(self, data_dir=None, name=None,
               Whether to send messages to the OCS publisher.
         payload_size : int, optional, default 2048
               The payload size to set on setup.
+        data_path_id : str, optional
```

Comment:
Sorry this is dumb, but could you change the docstring to say `data_path_id : str, optional, default None`?  Just to follow docstring convention.